README.md

```diff
@@ -14,99 +15,186 @@ Keep reading to get started or check out the [Roadmap](#roadmap) to see the feat
 
 ## Getting Started
 
-1. Clone this repo, `cd` into it, and `yarn install`
+### Step 1
 
-```
+Clone this repo, `cd` into it, and `yarn install`
+
+```zsh
 git clone git@github.com:redwoodjs/example-store-stripe.git
-cd example-store
+cd example-store-stripe
 yarn install
 ```
 
-2. Get your Stripe test keys
+### Step 2
+
+Get your Stripe test keys
 
 To develop on this repo locally, you'll need to populate your `.env` file with a few env vars. The first of those is your Stripe test keys.
 
-You'll need a Stripe account to get your test keys.
-If you don't already have one, you can make one here: https://dashboard.stripe.com/login?redirect=%2Ftest%2Fdashboard.
+First time visitor gets greeted like:
 
-Once you've made your account, if you weren't automatically redirected, navigate to your [test dashboard](https://dashboard.stripe.com/test/dashboard). You'll find your test keys on the right side of the page:
+<p align="center">
+<img width="400" alt="image" src="https://user-images.githubusercontent.com/2712405/173251273-00b09fa6-6b3a-40ac-9e94-48c6c4d4584d.png"/>
+<br/>
+<b>Image 1</b>
+</p>
+<br/>
 
-![image](https://user-images.githubusercontent.com/32992335/143495019-3c6319d3-f793-48c9-86ca-72f4c12f0306.png)
+Note the text "Start here" with an arrow that points to the button **`Start now >`**.
```

Comment:
This seems a bit like overkill to me; there's no way users could miss the "start here" text, it's huge.
```suggestion
```